### ISSUE TITLE
Are Period objects intended to be serializable?

### ISSUE THREAD

**Issue by guziy** [sdispater/pendulum#79]
Please consider the example below:

```python
In [1]: from pendulum import Period, Pendulum

In [2]: p1 = Pendulum.now()

In [3]: p2 = Pendulum.now()

In [4]: p1 == p2
Out[4]: False

In [5]: p1
Out[5]: <Pendulum [2016-11-16T15:32:16.211939-05:00]>

In [6]: p2
Out[6]: <Pendulum [2016-11-16T15:32:25.068809-05:00]>

In [7]: period = Period(p1, p2)

In [8]: period
Out[8]: <Period [2016-11-16T15:32:16.211939-05:00 -> 2016-11-16T15:32:25.068809-05:00]>

In [9]: import pickle

In [10]: pickle.dump(period, open("/HOME/huziy/period.bin", "wb"))

In [11]: period_restored = pickle.load(open("/HOME/huziy/period.bin", "rb"))
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-11-abdf2d0c5be1> in <module>()
----> 1 period_restored = pickle.load(open("/HOME/huziy/period.bin", "rb"))

/RESCUE/skynet3_rech1/huziy/python_virtualenvs/py3.4/lib/python3.4/site-packages/pendulum/period.py in __new__(cls, start, end, absolute)
     27
     28         return super(Period, cls).__new__(
---> 29             cls, seconds=delta.total_seconds()
     30         )
     31

AttributeError: 'int' object has no attribute 'total_seconds'
```

**Comment by sdispater** [comment]
Yes, you should be able to serialize a Period instance with `pickle`.

I will fix it.

**Comment by guziy** [comment]
I was curious trying to add ```__getnewargs__``` to the Period but in vain, still get integers as start and end parameters in the __new__ method

```python
 def __getnewargs__(self):
       return ((self.start, self.end), dict(absolute=self._absolute))
``` 

Cheers

**Comment by sdispater** [comment]
This has been fixed in commit 21edf3db12b4571717712fc9881407ce7da8a4ba.

It will land in the next `0.7.0` release

**Comment by guziy** [comment]
Cool,  ```__reduce__``` seems to be working better.

Cheers
